ui/src/theme.js

```diff
@@ -246,9 +234,30 @@ const createTheme = isDarkTheme => {
       },
     },
     overrides: {
+      MuiTypography: {
+        root: {
+          color: textPrimary,
+        },
+      },
+      MuiTableSortLabel: {
+        root: {
+          '&:hover': {
+            color: textPrimary,
+          },
+        },
+        icon: {
+          fontSize: '1rem',
+        },
+      },
+      MuiFormControlLabel: {
+        label: {
+          fontSize: '0.875rem',
+        },
```

Comment:
I wonder if this will be still be needed after removing the font-size from the body element (same thing with the icon class font-size).